f/fast-url-parser/fast-url-parser_ubi_8.6.sh

```diff
@@ -0,0 +1,39 @@
+#!/bin/bash -e
+# -----------------------------------------------------------------------------
+#
+# Package	: fast-url-parser
+# Version	: 1.1.3
+# Source repo	: https://github.com/apache/fast-url-parser
+# Tested on	: UBI 8.6
+# Language      : JavaScript
```

Comment:
we have been typically using "node" for nodejs packages , let's do the same for consistency

Reply:
Added changes.